### PR TITLE
update docs to describe usage of COLORS env variable

### PR DESCRIPTION
**Pull request by MynockSpit**
Update docs for https://github.com/testing-library/dom-testing-library/pull/1166.

NOTE: This is written in the same style as the docs for the other environment variable. I'd be happy to change one or both of those sections to some other format provided I'm given an example of a pattern to follow.

### CODE REVIEW

File: docs/dom-testing-library/api-debugging.mdx [hunk 1]
```diff
@@ -40,6 +40,20 @@ This works on macOS/Linux, you'll need to do something else for Windows. If
 you'd like a solution that works for both, see
 [`cross-env`](https://www.npmjs.com/package/cross-env).
 
+**Note**: The output of the DOM are colorized by default if you tests are
+running in a node environment. However, you may sometimes wish to turn off
+colors, such as in cases where the output is being written to a log file for
+debugging purposes. You can use the environment variable `COLORS` to explicitly
+force the colorization off or on. For example:
```
Comment (timdeschryver) [inline docs/dom-testing-library/api-debugging.mdx:47]:
```suggestion
**Note**: The output of the DOM is colorized by default if your tests are
running in a node environment. However, you may sometimes want to turn off
colors, such as in cases where the output is written to a log file for
debugging purposes. You can use the environment variable `COLORS` to explicitly
force the colorization off or on. For example:
```

File: docs/dom-testing-library/api-debugging.mdx [hunk 2]
```diff
@@ -40,6 +40,20 @@ This works on macOS/Linux, you'll need to do something else for Windows. If
...
+
+```
+COLORS=false npm test
+```
+
+As with `DEBUG_PRINT_LIMIT`, `COLORS` works in macOS/Linux but you'll need to do
+something else for Windows. If you'd like a solution that works for both, see
+[`cross-env`](https://www.npmjs.com/package/cross-env).
```
Comment (timdeschryver) [inline docs/dom-testing-library/api-debugging.mdx:55]: Sorry for the additional comment.
I think we can just copy paste this note, what do you think?

```suggestion
This works on macOS/Linux, you'll need to do something else for Windows. If
you'd like a solution that works for both, see
[`cross-env`](https://www.npmjs.com/package/cross-env).
```

Comment (MynockSpit) [reply docs/dom-testing-library/api-debugging.mdx:55]: Yeah, sure.

**Review by timdeschryver — APPROVED** [review]
Thanks!

**Review by eps1lon — APPROVED** [review]
Thanks!

**Comment by eps1lon** [comment]
@all-contributors add @MynockSpit for docs

**Pull request merged**